src/UI/styles/autofill-tooltip-styles.css

```diff
@@ -376,3 +396,51 @@ hr:first-child {
     right: 12px;
     top: 12px;
 }
+
+/* Import promotion prompt styles */
+
+.wrapper--credentials-import {
+    display: flex;
```

Comment:
I will debug this in the browser and see if there's opportunity to simplify some of this CSS. The layout shouldn't need this much CSS, given how close it is to the existing designs. But maybe I'm wrong 🙂. It's easier to see in the browser, so I'll spend some time to fire this up.

Reply:
I am not too sure, it was an interesting challenge to keep things separated causing some extra css. The Data tooltip is quite already overloaded, and has a bunch of logic, didn't want to abuse reuse here.

Reply:
My thought is that we should find a way to component-ize these tooltips, so they can contain their own styles e.g

Reply:
Removed almost all css classes, and added only one for the icon.